circle.yml

```diff
@@ -7,10 +7,6 @@ dependencies:
   override:
     - pip install -U pip
     - pip install -r requirements/dev.txt
```

Comment:
Need to install the requirements from test.txt for tests to run.  (Or else merge them into dev.in/txt, I'd be just as happy with that solution.)

Reply:
I just sent you a message about this.

Reply:
done